pkgs/top-level/all-packages.nix

```diff
@@ -18116,7 +18116,7 @@ with pkgs;
   in     if linker == "lld"     then llvmPackages.bintools-unwrapped
     else if linker == "cctools" then darwin.binutils-unwrapped
     else if linker == "bfd"     then binutils-unwrapped
-    else if linker == "gold"    then binutils-unwrapped
+    else if linker == "gold"    then binutils-unwrapped.override { enableGoldDefault = true; }
```

Comment:
can't test with

```
import ./. { localSystem = { system = "x86_64-linux"; }; crossSystem = { system = "x86_64-linux"; linker = "gold"; }; }
```

fails with the normal localSystem != crossSystem issue `> configure: error: C compiler cannot create executables`

Reply:
oh i can just do 
`:b (import ./. { localSystem = { system = "x86_64-linux"; }; crossSystem = { system = "aarch64-linux"; linker = "gold"; }; }).bash`

Reply:
`aarch64-unknown-linux-gnu-stage-final-gcc> checking whether we are using gold... yes`